recipe/yum_requirements.txt

```diff
@@ -1,6 +1,6 @@
 csh
 imake
-libx11-devel
+libX11-devel
```

Comment:
That is needed because `yum_requirements.txt` is a hack that adds the extra dependencies needed in the CI configuration. See https://github.com/conda-forge/ncl-feedstock/blob/master/ci_support/run_docker_build.sh#L49 for example.